src/store/types/geocodingReducer.types.d.ts

```diff
@@ -0,0 +1,50 @@
+export enum GeocodingActionTypes {
+  ADD_TO_STORY = 'ADD_TO_STORY',
+  REMOVE_FROM_STORY = 'REMOVE_FROM_STORY',
+  CLEAR_STORY = 'CLEAR_STORY',
+  FETCH_STORY = 'FETCH_STORY',
+}
+
+export interface IGeocodingItem {
+  place_id: number;
+  licence: string;
+  powered_by: string;
+  osm_type: string;
+  osm_id: number;
+  boundingbox: {
+    '0': string;
+    '1': string;
+    '2': string;
+    '3': string;
+  };
```

Comment:
please extract this to the separate type

Reply:
@ufoBeliever please take action

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/37/commits/9c3bff5628ec16686f9f6220ae73995d9479c4da